TypeCobol.Test/Parser/CodeElements/STOP.txt

```diff
@@ -1,4 +1,22 @@
 --- Diagnostics ---
+Line 1[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 5[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 6[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 7[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 8[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 9[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 10[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 11[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 12[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 13[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 14[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 15[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 16[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 17[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 18[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 19[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 20[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
+Line 22[1,4] <37, Warning, General> - Warning: GOBACK should be used instead of STOP RUN RuleStack=codeElement>stopStatement,  OffendingSymbol=[1,4:STOP]<STOP>
```

Comment:
The warning should be displayed only on keywords "STOP RUN" not on any "STOP (something)"

Reply:
fixed 23/09/2019
in class StopStatementChecker with the method OnCodeElement